requirements-dev.txt

```diff
@@ -50,6 +53,7 @@ pydocstyle==6.3.0
 pyfakefs==5.4.1
 pyflakes==3.2.0
 pygit2==1.14.1
+pylint==3.2.6
```

Comment:
You'll probably want to add these to the pyproject.toml file if tics needs them, or we'll end up with the next run of `freeze-requirements.sh` removing them

Reply:
done!